toolkits/nature/packages/nature-sort-by/README.md

```diff
@@ -25,34 +25,17 @@ sortBy();
 ```scss
 @import '@springernature/nature-sort-by/scss/10-settings/sort-by';
 @import '@springernature/nature-sort-by/scss/50-components/sort-by';
-
-@import '@springernature/brand-context/default/scss/60-utilities/flex.scss';
-@import '@springernature/brand-context/default/scss/60-utilities/hiding.scss';
-@import '@springernature/brand-context/default/scss/60-utilities/spacing.scss';
-
 ```
 
-> **NOTE** The component require the use of the utility classes shown above
+The component's HTML should be inserted inside of an HTML form in your application. See below [Template section](#template) for an example.
```

Comment:
Try to use active, rather than passive language where you can, as it's clearer for people to read and understand. Happy to talk through the difference if that's useful :)

```suggestion
Insert the component's HTML inside an HTML form in your application. See an example [template](#template) below.
```